project_task_category/migrations/10.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2018 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_model_renames = [('project.category.main', 'project.category'), ]
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    cr = env.cr
+    openupgrade.lift_constraints(cr, 'project_task', 'categ_id')
+    openupgrade.lift_constraints(cr,
+                                 'business_requirement_resource_template',
```

Comment:
I think there is a mismatch here: this should be in the business requirement related module. not here.